### DISCUSSION TITLE
The latest commit is so fast!

### DISCUSSION THREAD

**Discussion by greatmate98** [UltimaHoarder/UltimaScraper#1418] [General]
Just wanted to show my appreciation for the improvements in speed in the latest commit.  I don't know if ayncio was the secret sauce, but dam it's blazing fast now!!  Incredible work.  I felt it was getting slower and slower each update from the point you get a confirmed login to the point you can actually select a model, then from the time you select a model and the downloads actually start.  My internet isn't the best mind you...but this latest commit seems to me to be at least a 10-20 x improvement in speed for everything except the downloading part, which was already as fast as my internet could possibly achieve.  Hats off.

**Comment by UltimaHoarder** [comment]
Thanks mane, and yes it was because of asyncio :)

There's still more work to do though.

**Comment by ghost** [comment]
I agree just switched to the new commit today and damn it is fast! I also like the colors and the speed now being in one line and not breaking into a new line :)
